lntest/rpc/harness_rpc.go

```diff
@@ -84,6 +84,11 @@ func NewHarnessRPC(ctxt context.Context, t *testing.T, c *grpc.ClientConn,
 	return h
 }
 
+// Context returns the context of the RPC harness.
+func (h *HarnessRPC) Context() context.Context {
```

Comment:
I think we can just use `context.Background()` in the test, so we don't need to expose this.
The important part is that we have a timeout on RPC calls.